test/scripts/rpm-ostree_mock

```diff
@@ -5,8 +5,8 @@
 # Write invocation with arguments to a file to allow making assertion.
 echo "$*" >> /rpm-ostree_calls
 
-if [ "$*" != *"--append"* ]
-then
+# if [ "$*" != *"--append"* ]
```

Comment:
@SchSeba if it's fine with you, I'll just remove the conditional, as a test relies on the broken behavior as you can see in [this run](https://github.com/k8snetworkplumbingwg/sriov-network-operator/actions/runs/11053030453/job/30706442667).

Reply:
@zeeke can you have a look at this please

Reply:
With [this change](https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/781/commits/5d2415e28a18acc6e886b152db7d3c04184182cb), the test fail with:

```
+ chroot /host/ rpm-ostree kargs
+ /usr/sbin/chroot /tmp/tmp.gOQJTP0gp3 rpm-ostree kargs
+ grep -vq X=Y
/usr/sbin/chroot: failed to run command ‘rpm-ostree’: No such file or directory
```

Which doesn't really make sense to me. I'll try changing the shebang back to `#!/usr/bin/sh` to see if it makes a difference.

https://github.com/k8snetworkplumbingwg/sriov-network-operator/actions/runs/11065756907/job/30745730017?pr=781#step:9:105

Reply:
OK, https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/781/commits/a5c53b587e521cfe7d874516e2b7624b4ee01258 worked.

I think this is ready for reviews now.

Reply:
hi, from [this run](https://github.com/k8snetworkplumbingwg/sriov-network-operator/actions/runs/11066086546/job/30746657984?pr=781) I see:
```
+ grep -q \--append
/usr/bin/rpm-ostree: 8: grep: not found
```

can you add `grep` to the command list [here](https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/781/files#diff-3edef0a0ec747a6f82bd0f9c61538add1ca891ca37c7e85b9fd46d46b343d848L42)?